src/client/metric.rs

```diff
@@ -107,6 +99,35 @@ pub enum Space {
     EByte
 }
 
+impl Space {
+    fn from_u8(x: u8) -> Option<Self> {
+        match x {
+            0 => Some(Space::Byte),
```

Comment:
@ryandoyle  these are only convenience functions to create Units from their PMAPI representations (unsigned integers, as enums values map to them in C), however @saurvs has implemented the API more in tao of rust than C, and enums are first class values. This function is just a convenience that returns an enum value from an integer. He's also impl `fmt::Display` so you can directly print a `Space::Byte` as `KiB` for debugging